Mastodon/Scene/HashtagTimeline/HashtagTimelineHeaderViewActionButton.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  HashtagTimelineHeaderViewActionButton.swift
+//  Mastodon
+//
+//  Created by Marcus Kida on 25.11.22.
+//
+
+import UIKit
+import MastodonUI
+import MastodonAsset
+
+class HashtagTimelineHeaderViewActionButton: RoundedEdgesButton {
```

Comment:
"unfollow" looks kind of weird in dark mode? Also in light mode, it looks kind of ... inactive? You might need to toggle appearance a bit to see that.

<img width="105" alt="Screenshot 2022-11-30 at 18 07 53" src="https://user-images.githubusercontent.com/2580019/204862557-ddf5c39a-e0d8-4ec6-95f6-c2e0cab0f1db.png">
<img width="108" alt="Screenshot 2022-11-30 at 18 09 31" src="https://user-images.githubusercontent.com/2580019/204862818-8e9f5535-f98e-4fcc-add7-89372606ff4f.png">

TBH: I haven't checked the design 🙈

Reply:
Hm you're right, might have missed this in dark mode. IIRC there were ne designs for dark mode, but I'll fix it and improve this. Thanks for checking!

Reply:
@zeitschlag FYI 👆

Reply:
Looks great 👍